spec/factories/users.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :user do
+    first_name {Faker::Name.last_name}
+    last_name {Faker::Name.last_name}
+    email {Faker::Internet.email}
+    user_name {Faker::Name.first_name}
+    phone {Faker::PhoneNumber.cell_phone_in_e164}
+    encrypted_password{Faker::Internet.password(min_length: 10, max_length: 20)}
```

Comment:
Can you please check bcrypt gem? I am providing a reference. Thank you!
https://medium.com/justinctollison/bcrypt-gem-and-password-hashing-edf6ae1c69a2

Reply:
I used bcrypt while implementing manual authentication . should I use it to hash password in factory bot?